### DISCUSSION TITLE
Browserslist: caniuse-lite is outdated.

### DISCUSSION THREAD

**Discussion by bgznb** [NervJS/taro#11536] [Q & A]
npx browserslist@latest --update-db 运行了这个也没用,大家都是怎么解决的呢? node_modules 文件夹删了重装也没用

**Comment by guohuihot** [comment]
同样的问题，应该是npm包里哪个依赖需要更新了

**Comment by yoyo837** [comment]
不是自己依赖的就无视吧, 某个依赖包依赖的, 强迫症的话也先忍着. 或者用类似`resolution`等方式解决.

**Reply by yoyo837** [reply]
实际上他就不需要解决, 就好比你手机上软件老提醒你更新版本一样.
